From 60ab568ec9dc3046a5ab9ec25bfb467bc0f7567a Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Thu, 28 Nov 2024 15:14:18 +0800
Subject: [PATCH 1/1] quirks-add-pressure-pad-quirk-for-Lenovo-ThinkBook-2
Some Lenovo ThinkBook 14 G6+ IMH models ship with pressure pads (nominally
"Force Pad")
---
 quirks/50-system-lenovo.quirks | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/quirks/50-system-lenovo.quirks b/quirks/50-system-lenovo.quirks
index 1582542..f6da2cc 100644
--- a/quirks/50-system-lenovo.quirks
+++ b/quirks/50-system-lenovo.quirks
@@ -384,3 +384,10 @@ MatchName=*GXTP5100*
 MatchDMIModalias=dmi:*svnLENOVO:*pvrThinkBook14G7+ASP*:*
 MatchUdevType=touchpad
 ModelPressurePad=1
+
+# The ThinkBook 14 G6+ also has this issue
+[Lenovo ThinkBook 14 G6+ IMH]
+MatchName=*GXTP5100*
+MatchDMIModalias=dmi:*svnLENOVO:*pvrThinkBook14G6+IMH*:*
+MatchUdevType=touchpad
+ModelPressurePad=1
-- 
2.45.2
